bazel/repository_locations.bzl

```diff
@@ -5,7 +5,7 @@ REPOSITORY_LOCATIONS = dict(
         remote = "https://github.com/google/boringssl",
     ),
     com_google_absl = dict(
-        commit = "bd5a2bba8f4238db2e06c99ac808fc8327dec2a0",  # 2018-05-07
+        commit = "26b789f9a53d086c8b8c9c2668efb251e37861cd",  # 2018-05-04
```

Comment:
Is there an upstream bug for abseil? We should probably reference it in the PR and here in a comment, to avoid having the next person come in here and inadvertently revert the change.

Reply:
Upstream bug is referenced in the commit message. I'll add a comment in this file as well.